src/site/content/en/progressive-web-apps/pwa-checklist/index.md

```diff
@@ -0,0 +1,482 @@
+then fall back to cached content or an offline indicator as needed.
+
+{% endDetails %}
+
+
+{% Details %}
+{% DetailsSummary 'h3' %}
+
+Is fully accessible
+
+All user interactions pass [WCAG 2.0](https://www.w3.org/TR/WCAG20/)
+accessibility requirements.
+
+{% endDetailsSummary %}
+
+All user interactions pass [WCAG 2.0](https://www.w3.org/TR/WCAG20/)
+accessibility requirements.
+
+#### Why
+
+Most people at some point in their life will want to take advantage of your
+PWA in a way that is covered under the [WCAG 2.0](https://www.w3.org/TR/WCAG20/)
+accessibility requirements. Humans' ability to interact with and understand
+your PWA exist on a spectrum and needs can be temporary or permanent. By
+making your PWA accessible, you ensure it's usable for everyone.
+
+#### How
+
+W3C's [Introduction to Web Accessibility](https://www.w3.org/WAI/fundamentals/accessibility-intro/)
+is a good place to start. For testing, a majority of accessibility testing
+must be done manually. Tools like the [Accessibility](/lighthouse-accessibility/)
+audits in Lighthouse, [axe](https://github.com/dequelabs/axe-core), and
+[Accessibility Insights](https://accessibilityinsights.io/) can help you
+automate some accessibility testing. It's also important to use semantically
+correct elements instead of recreating those elements on your own
+(for instance, `a` and `button` elements). This ensures that when you do need
+to build more advanced functionality, accessible expectations are met (like
+when to use arrows vs. tabs).
+[A11Y Nutrition Cards](https://accessibilityinsights.io/) has excellent
+advice on this for some common components.
+
+{% endDetails %}
+
+
+{% Details %}
+{% DetailsSummary 'h3' %}
+
+Can be discovered through search
+
+Your PWA can be easily [discovered through search](/discoverable/).
+
+{% endDetailsSummary %}
+
+Your PWA can be easily [discovered through search](/discoverable/).
+
+#### Why
+
+One of the biggest advantages of the web is the ability to discover sites and
+apps through search. Making sure that canonical URLs exist for content and
+that search engines can index your site is critical for users to be able to
+find your PWA. This is especially true when adopting client-side rendering.
+
+#### How
+
+You can use [Google Search Console](https://search.google.com/search-console/about)
+and the [Search Engine Optimization audits](/lighthouse-seo/) in Lighthouse to
+help you debug and fix discoverability issues with your PWA. You can also use
+[Bing](https://www.bing.com/toolbox/webmaster)'s or
+[Yandex](https://webmaster.yandex.com/welcome/)'s webmaster tools, and
+consider including [structured data](https://goo.gle/search-gallery) via
+schemas from [Schema.org](https://schema.org/) in your PWA.
+
+{% endDetails %}
+
+
+{% Details %}
+{% DetailsSummary 'h3' %}
+
+Works with any input type
+
+Your PWA is equally usable with a mouse as it is with a keyboard, by touch,
+or with a stylus.
```

Comment:
Change to:

Your PWA is equally usable with a mouse, a keyboard, a stylus, or touch.